Makefile

```diff
@@ -163,6 +163,14 @@ _opt_pylint:
 	@command -v flake8 && { $(MAKE) pylint; exit $$?; } \
 		|| echo "SKIP: pylint (flake8 not found)"
 
+commitlint:
+	# $(realpath build/bin/nvim) -es -u NONE +"luafile scripts/lintcommit.lua"
+	$(realpath build/bin/nvim) -u NONE +"luafile scripts/lintcommit.lua"
```

Comment:
Note: `realpath` doesn't exist by default on macOS, you need to install it with `coreutils`. All macOS devs should probably have that, well one would hope so at least, but it may currently cause problems if you want to use it in the CI: https://github.com/neovim/neovim/blob/1f49268c46fcbe65f7e2e2cb620e6f51c059cf9e/.github/workflows/release.yml#L86

Reply:
I've decided to split this PR into two parts. This one I'll only focus on making the linter work locally and the next part I'll focus on making it work with the CI. But I'll keep this in mind for the next part, thank you.